src/library/Uno.Material/Styles/Controls/v2/FloatingActionButton.xaml

```diff
@@ -1,10 +1,10 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-					xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-					xmlns:toolkit="using:Uno.UI.Toolkit"
-					xmlns:um="using:Uno.Material"
-					mc:Ignorable="d">
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+                    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+                    xmlns:toolkit="using:Uno.UI.Toolkit"
+                    xmlns:um="using:Uno.Material"
+                    mc:Ignorable="d">
```

Comment:
Still a good amount of places that are spaces instead of tabs :)

Reply:
my mistake, it's fixed now